lib/core/include/irods_plugin_base.hpp

```diff
@@ -293,6 +293,48 @@ namespace irods
             }
         } // call
 
+        template <typename... Args>
+        error call_without_policy(
+            rsComm_t*                     _comm,
+            const std::string&            _operation_name,
+            irods::first_class_object_ptr _fco,
+            Args...                       _args)
+        {
+            try {
+                plugin_context ctx(_comm, properties_, _fco, "");
+#if 0
```

Comment:
suspicious

Reply:
this is temporary. this block will be removed if these changes pass CI.